src/Sarif.Driver/Sdk/CommonOptionsBase.cs

```diff
@@ -19,15 +20,23 @@ public class MultitoolOptionsBase
             "force",
             Default = false,
             HelpText = "Force overwrite of output file if it exists.")]
-        public bool Force { get; internal set; }
+        public bool Force { get; set; }
```

Comment:
Ok, I never found out. Why does this need a public setter?

---
In reply to: [214519998](https://github.com/Microsoft/sarif-sdk/pull/1062#discussion_r214519998) [](ancestors = 214519998)

Reply:
You mean internal.  These are indicators that unit tests are creating/initializing these object (which are otherwise constructed only by the commandlineparser library). We could skip these hijinks by the expedient of a ctor to init...)

---
In reply to: [214521824](https://github.com/Microsoft/sarif-sdk/pull/1062#discussion_r214521824) [](ancestors = 214521824,214519998)